packer/securing-docker/README.md

```diff
@@ -0,0 +1,68 @@
+
+#### Requirements 
+
+- [Packer](https://www.packer.io/intro/getting-started/install.html)
+- [AWS account](https://aws.amazon.com/) - free tier will work with our examples.
+- [Security Credentials](https://console.aws.amazon.com/iam/home) from your AWS account
+
+#### Setup
+ 
+ 1. Setup packer
+ 2. `$ git clone git@github.com:nearform/devops.git`
+ 3. Create a file named `variables.json` in the root of the example repo from #2.  Add your access and secret keys. If you want to use a different region in AWS change the `aws_region` and find replace the `ubuntu_source_ami` with the AMI ID that has an instance type of `hvm-ssd` from [Ubuntu](https://cloud-images.ubuntu.com/locator/).  Your File should look something like this:
+ 
+``` json
+{
+  "aws_secret_key": "< INSERT YOUR SECRET KEY >",
+  "aws_access_key": "< INSERT YOUR ACCESS KEY >",
+  "aws_region": "us-east-1",
+  "ubuntu_source_ami": "ami-cb1d41b0"
+}
+```
+ 
+ #### Build the AMI
+ After executing the command below you should see an AMI in the [AWS Console](https://console.aws.amazon.com/ec2/v2/home?#Images)
+ 
+ `$ packer build -var-file=variables.json base-image-us-east-template.json`
+
+Our example template uses packer's [amazon-ebs](https://www.packer.io/docs/builders/amazon-ebs.html) builder.  We use a builder and two provisioners in our template to accomplish the following:
+
+1. Create /etc/docker/daemon.json
+2. Create and mount the volume that will be used to store our Docker assets
+3. Install Docker CE
+4. Clone [Docker-Bench-Security](https://github.com/docker/docker-bench-security) 
+5. Install and configure auditd
+
+#### Run an Instance of Your AMI
+Once packer has finished building your AMI, you can log into the AWS Console and launch an instance from the AMI.  You will only need to configure network related items (i.e. subnet) and the rest you should be able to take the defaults. Once your instance is up and running ssh into it and you should find docker-bench-security in the home directory of the ubuntu user. Do the following to run and verify the AMI passes the test we've documented above.
+
+1. `$ cd docker-bench-security`
+2. `$ sudo sh ./docker-bench-security.sh `
+ 
+*note*: sudo is used in step 2 due to checks against files and directories that require elevated privileges.
+
+#### Run a Container in your Instance
+If you would like to build a docker image and run a container based on that image you can include [./files/app/]() in your AMI build, which will give you a simple [hapi](https://hapijs.com/) app to run the [Docker-Bench-Security](https://github.com/docker/docker-bench-security) tests against.
+
+To include these files add the following to the provisioners section of the template
+
+```
+    {
+      "type": "shell",
+      "inline": ["mkdir ./app/"]
+    },   
+    {
+      "type": "file",
+      "source": "./files/app/",
+      "destination": "./app/"
+    }
+```
```

Comment:
yeah I think coming back in with a link to the blog post would help once that has landed.